examples/demo/gloo/molecular_viewer.py

```diff
@@ -74,7 +74,7 @@
     pos.z += v_radius*z;
     vec3 pos2 = pos.xyz;
     pos = u_projection * pos;
-    gl_FragDepth = 0.5*(pos.z / pos.w)+0.5;
+//    gl_FragDepth = 0.5*(pos.z / pos.w)+0.5;
```

Comment:
The gl_FragDepth command doesn't work in WebGL. It was commented out earlier in the corresponding iPython example, https://github.com/vispy/vispy/blob/master/examples/ipynb/webgl_example_2.ipynb#L102.

We commented it because we're not sure if we want to keep things that fail in WebGL.